src/components/Dialogs/WorkspaceDialog/WorkspaceDialogComponent.tsx

```diff
@@ -67,6 +67,12 @@ export const WorkspaceDialogComponent = observer(() => {
                 return;
             }
 
+            // TODO: to be removed after storing SystemType in workspace
+            if (appStore.overlayStore.isImgCoordinates && appStore.frames.map(frame => frame.spatialReference !== null).includes(true)) {
+                AlertStore.Instance.showAlert("Not supporting save workspace with spatial matching in image cooordinates");
```

Comment:
Might be more explicit if we add: "Saving workspace failed: ..." to show that the workspace is not saved (instead of saved but with matching disabled).

Reply:
has changed!